examples/suave/bids/index.ts

```diff
@@ -65,7 +65,7 @@ export class MevShareBid {
     return {
       to: this.mevShareContract,
       data: this.newBidCalldata(),
-      type: SuaveTxTypes.ConfidentialRequest,
+      type: '0x43',
```

Comment:
why replace with hard-coded value instead of const?

Reply:
you can use either one -- this is just more convenient, and a demonstration of what most people will likely do